applications/CompressiblePotentialFlowApplication/python_scripts/initialize_geometry.py

```diff
@@ -0,0 +1,252 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CompressiblePotentialFlow
+import KratosMultiphysics.MeshingApplication as MeshingApplication
+import math
+import time
+
+
+def Factory(settings, Model):
+    if( not isinstance(settings,KratosMultiphysics.Parameters) ):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return InitializeGeometryProcess(Model, settings["Parameters"])
+def MoveModelPart(origin, skin_model_part):
+    for node in skin_model_part.Nodes:
+        node.X=origin[0]+node.X
+        node.Y=origin[1]+node.Y
+def RotateModelPart(origin, angle, model_part):
+    ox,oy=origin
+    for node in model_part.Nodes:
+        node.X = ox+math.cos(angle)*(node.X - ox)-math.sin(angle)*(node.Y - oy)
+        node.Y = oy+math.sin(angle)*(node.X - ox)+math.cos(angle)*(node.Y - oy)
+## All the processes python should be derived from "Process"
+class InitializeGeometryProcess(KratosMultiphysics.Process):
```

Comment:
I find that the name of this class is misleading and not self-descriptive enough. I'd suggest something like `LevelSetBasedRemeshingProcess`.

By the way, is this compatible with any continuous level-set based formulation?

Reply:
> I find that the name of this class is misleading and not self-descriptive enough. I'd suggest something like `LevelSetBasedRemeshingProcess`.

I agree that name is improvable, but your suggestion would be also incomplete. This process, as it is right now, it's used even if there is no remeshing (note that there's a flag that skips the remeshing loop to just initialize the geometry).

```
        self.InitializeSkinModelPart()
        self.CalculateDistance()
        ini_time=time.time()
        while self.step < self.max_iter and self.do_remeshing:
            self.step += 1
            self.ExtendDistance()
            self.RefineMesh()
            self.CalculateDistance()
            self.UpdateParameters()
        self.ModifyFinalDistance()
        self.ExtendDistance()
        self.CopyAndDeleteDefaultDistance()
```
We could separate this loop in three files:
- initialize_skin_to_model_part_process.py (containg move_skin_model_part, rotate_skin_model_part)
- level_set_based_remeshing_process.py
- modify_distance_process.py

Any other suggestion?
> 
> By the way, is this compatible with any continuous level-set based formulation?

Yes, I would say it is!

Reply:
If what you've done is compatible with any continuous level-set based formulation, it might have sense to separate it in a different utilities, something that would go in line to what @loumalouomega is trying to do in #4558 .

May be it's worth it to discuss it...

Reply:
Actually, taking a look at https://github.com/KratosMultiphysics/Kratos/pull/4558#issuecomment-478496561, I'm not sure if what im doing here would work if the skin is moving at each time step. 

@loumalouomega When you can, could you take a look at my scritps? I see you are defining new AnalysisStages to do the remeshing. Is that also for the distance case (when we only refine with resepct to a distance)?